nvm.sh

```diff
@@ -2345,16 +2345,11 @@ nvm() {
     return $?
   fi
 
-  local COMMAND
-  COMMAND="${1-}"
-  shift
-
-  # initialize local variables
-  local VERSION
-  local ADDITIONAL_PARAMETERS
-
-  case $COMMAND in
-    'help' | '--help')
+  argsList=("$@")
+  for i in ${argsList[@]}
+  do
+    case $i in
+      'help'|'-help'|'--help')
```

Comment:
I'm a bit surprised to see `nvm -help` as an alias for `nvm --help` (i'd totally forgotten about `nvm help`). Why the extra alias? (typically, single-dash arguments are always only one letter; only double-dash arguments are longer words)

Reply:
In case someone were to type one dash either because they are not too familiar with nvm/shell or because it's a typo, then this would catch this variation.

Reply:
@ljharb the extra `-help` is now `-h` and the test adjusted for it.